src/core_plugins/kibana/public/discover/index.html

```diff
@@ -30,12 +30,19 @@
               input-focus
               kbn-typeahead-input
               ng-model="state.query"
-              placeholder="Search..."
+              placeholder="Search... e.g. status:200 AND extension:PHP"
```

Comment:
Minor feedback, take it or leave it: I think it might help to throw in parentheses here so it's "Search... (e.g. status:200 AND extension:PHP)" but that's just my preference.